client/my-sites/site-settings/reading-newsletter-settings/ExcerptSetting.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslate } from 'i18n-calypso';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+
+type ExcerptSettingProps = {
+	value?: boolean;
+	disabled?: boolean;
+	updateFields?: ( fields: { [ key: string ]: unknown } ) => void;
+};
+
+export const ExcerptSetting = ( {
+	value = false,
+	disabled,
+	updateFields,
+}: ExcerptSettingProps ) => {
+	const translate = useTranslate();
+	return (
+		<FormFieldset>
+			<FormLabel>For each new post email, include</FormLabel>
+			<FormLabel>
+				{ /* @ts-expect-error FormRadio is not typed and is causing errors */ }
+				<FormRadio
+					checked={ ! value }
+					onChange={ () => updateFields?.( { wpcom_subscription_emails_use_excerpt: false } ) }
+					disabled={ disabled }
+					label={ translate( 'Full text' ) }
+				/>
+			</FormLabel>
+			<FormLabel>
+				{ /* @ts-expect-error FormRadio is not typed and is causing errors */ }
+				<FormRadio
+					checked={ value }
+					onChange={ () => updateFields?.( { wpcom_subscription_emails_use_excerpt: true } ) }
+					disabled={ disabled }
+					label={ translate( 'Excerpt' ) }
+				/>
+			</FormLabel>
+			<FormSettingExplanation>
+				{ translate(
+					'Sets whether email subscribers can read full posts in emails or just an excerpt and link to the full version of the post. Learn more about sending emails.'
+				) }
```

Comment:
We are missing the link to the support page here (as discussed in https://github.com/Automattic/wp-calypso/issues/70427#issuecomment-1329737972). Here's the suggested code (plus, we will need that `import { localizeUrl } from '@automattic/i18n-utils';` import):

```suggestion
				{ translate(
					"Sets whether email subscribers can read full posts in emails or just an excerpt and link to the full version of the post. {{link}}Learn more about sending emails.{{/link}}.",
					{
						components: {
							link: (
								<a
									href={ localizeUrl( 'https://wordpress.com/support/launch-a-newsletter/' ) }
									target="_blank"
									rel="noreferrer"
								/>
							),
						},
					}
				) }
```

Reply:
Nice! Thanks for catching this. I created a draft PR for this before the weekend, and I forgot what I already implemented and what else needed to be done. Thanks for catching this again! 🦅

Reply:
My pleasure! The changes look good to me. 👍🏼 I will give it one final test tomorrow morning and approve the PR. 🙂